### DISCUSSION TITLE
Homebridge 2.0

### DISCUSSION THREAD

**Discussion by thedovester101** [iRayanKhan/homebridge-tuya#490] [New device help]
Hello, Will this/is this plugin compatible with Homebridge 2.0?

**Comment by pespinel** [comment]
HomebridgeV2 will be released soon and this plugin is not marked as compatible. The maintainers should review it:

https://github.com/homebridge/homebridge/wiki/Updating-To-Homebridge-v2.0#for-plugin-developers

![image](https://github.com/user-attachments/assets/26019690-26db-4007-a03a-8b776a266f4e)
